cmd/vicadmin/vicadm.go

```diff
@@ -434,7 +434,9 @@ func (r datastoreReader) open() (entry, error) {
 	}
 
 	req, err := http.NewRequest("GET", u.String(), nil)
-	req.AddCookie(ticket)
+	if ticket != nil {
```

Comment:
I noticed that too, but left it alone for now too keep the diff focused.

I think we should just s/err/_/ here, since the url has already been parsed and we know "GET" is a valid method - the only 2 cases where it would return in an error.